network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/AbstractIdentifiableImpl.java

```diff
@@ -43,18 +45,51 @@ public abstract class AbstractIdentifiableImpl<I extends Identifiable<I>, D exte
     // Needed to generate the exception message when accessing a removed identifiable
     private String idBeforeRemoval;
 
+    private static final String PROPERTIES = "properties";
+
     protected AbstractIdentifiableImpl(NetworkObjectIndex index, Resource<D> resource) {
         this.index = index;
         this.resource = resource;
     }
 
     public void updateResource(Consumer<Resource<D>> modifier) {
```

Comment:
```suggestion
    public void updateResourceWithoutNotification(Consumer<Resource<D>> modifier) {
```
to make it clearer that it does not notify?

Reply:
:+1: 
Renamed